packages/domainproperties/src/models.ts

```diff
@@ -318,6 +318,15 @@ export class DomainDesign extends Record({
     isNameSuffixMatch(name: string): boolean {
         return this.name && this.name.endsWith(name + ' Fields');
     }
+
+    isValid(): boolean {
+        const invalidField = this.fields.find((field) => !field.isValid());
```

Comment:
I think there is an "any" or "some" helper that you could likely user here instead of find. Then you could just return the equivalent of "not this.fields.any(!isValid)"...not sure on the syntax.

Reply:
Just saw the getInvalidFields() below, if you don't use the any/some approach, you could at least use that helper here.

Reply:
I ended up changing the implementation of this function.